rudder-web/src/main/scala/com/normation/rudder/web/rest/RestUtils.scala

```diff
@@ -94,7 +94,7 @@ object RestUtils extends Loggable {
    *   - Add a new line at the end and beginning of an array and indent one more level array data
    *   - space after colon
    */
-  def render(value: JValue): Document = {
+  def rendrrer(value: JValue): Document = {
```

Comment:
maybe it is used only at runtime ?
could you at least update the comment on top of the method to link to the actual version of the method in lift 2.6.3?